nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -181,6 +181,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `p4` package now only includes the open-source Perforce Helix Core command-line client and APIs. It no longer installs the unfree Helix Core Server binaries `p4d`, `p4broker`, and `p4p`. To install the Helix Core Server binaries, use the `p4d` package instead.
 
+- The NextCloud NixOS module uses OpenSSL 3.x for its PHP's openssl extension, this breaks RC4-based server-side encryption in NextCloud, making all your files unreadable upon upgrade, no data loss **should occur** though. To restore functionality, `services.nextcloud.enableBrokenCiphersForSSE` has to be set to `true`. NextCloud is planning to implement AES-256-GCM server-side encryption in the future through <https://github.com/nextcloud/server/pull/25551>.
```

Comment:
Please link the option via `[](#opt-services.nextcloud.enableBrokenCiphersForSSE)`. Then people can directly navigate to the option's decription for further context.

Also, I'm wondering if we should check whether that's actually the case (I can do that this week).

Reply:
> I can do that this week

Just did it:

* with openssl3 for php installed, you can neither download nor upload files (as expected).
* you can disable encryption and upload new files (old files still inaccessible)
* you can install openssl1_1 and re-enable encryption and force decrypt all

IoW we can write something like "upon testing we couldn't trigger any cases of data-loss, but we cannot guarantee that for every accidental openssl upgrade".